bench/box2d/optimized/imgui_draw.ll

```diff
@@ -23606,8 +23606,10 @@ _ZL36ImFontAtlasBuildRenderDefaultTexDataP11ImFontAtlas.exit: ; preds = %._crit_
   br i1 %.not115.i, label %.preheader100.i, label %.lr.ph.preheader.i
 
 .lr.ph.preheader.i:                               ; preds = %159
-  %umax.i = zext nneg i32 %155 to i64
-  tail call void @llvm.memset.p0.i64(ptr noundef nonnull align 1 dereferenceable(1) %169, i8 0, i64 %umax.i, i1 false), !tbaa !23
+  %170 = add nsw i32 %155, -1
+  %171 = zext i32 %170 to i64
+  %172 = add nuw nsw i64 %171, 1
```

Comment:
missing fold to `%171 = zext i32 %155 to i64`, reduced test case in https://github.com/llvm/llvm-project/pull/159942/files#diff-633f5da0f58a9d1a0b5eba43fbe72040b9716f3270ee4dea9f5888b3b3c66074